openslides_backend/action/mixins/send_email_mixin.py

```diff
@@ -199,3 +199,20 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
                 raise ActionException(f"{self.check_email_field} must be valid email.")
         instance = super().update_instance(instance)
         return instance
+
+
+class EmailSenderCheckMixin(Action):
+    check_email_sender_field = "users_email_sender"
+    blacklist = ("[", "]", "\\")
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        if instance.get(self.check_email_sender_field):
+            if any(
+                entry in instance[self.check_email_sender_field]
+                for entry in self.blacklist
+            ):
+                raise ActionException(
+                    f"{self.check_email_sender_field} must not contain {', '.join(self.blacklist)}."
```

Comment:
Either just print the chars one after another or enclose them in `'`, else it looks confusing

Reply:
Changed the error message into `"users_email_sender must not contain '[', ']', '\\'."`